python/deptry/dependency_getter/builder.py

```diff
@@ -41,6 +43,15 @@ class DependencyGetterBuilder:
     def build(self) -> DependencyGetter:
         pyproject_toml_found = self._project_contains_pyproject_toml()
 
+        if not self.using_default_requirements_files:
+            if not self._any_requirements_files_exists():
+                raise FileNotFoundError(  # noqa: TRY003
```

Comment:
Added `noqa` here. Alternative of creating a class called `RequirementsFilesSpecifiedExplicitlyButNotFound` feels a bit superfluous 🙈